hack/functest.sh

```diff
@@ -1,3 +1,12 @@
 #!/bin/bash -e
 
-echo hello world
+source ./cluster/cluster.sh
+
+[ "$CI" == "true" ] && dnf install -y bind-utils
+
+if ! nslookup www.google.com > /dev/null 2>&1; then
+  echo "nslookup error"
+  exit 1
+fi
+
+KUBECONFIG=${KUBECONFIG:-$(cluster::kubeconfig)} $GO test -test.timeout=1h -test.v ./tests/... -ginkgo.v
```

Comment:
i think we need to add no-color
else we see the control characters on the log

Reply:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubesecondarydns/38/pull-kubesecondarydns-e2e-k8s/1599451475841388544
Can you please point me what control characters do we see here?

Reply:
We don't have the no-color flag in kubevirt. And I find it nice to see the colors. I prefer to keep it.

Reply:
automation/test.sh:ginko_params="--no-color --seed=42"

from kubevirt